README.md

```diff
@@ -102,6 +102,31 @@ cause degradation of performance.
 
 Without git revision specified HEAD is used.
 
+## Build Python package
+
+To create a binary "wheel" distribution, use:
+
+    python setup.py bdist_wheel
```

Comment:
Are there any specific benefits?  I've not seen this way before, but according to google, it can be fairly slow because it copies everything into a temp directory before doing the build.

Reply:
Moving forward, the idea is to build python package using a standard approach that is independent from the build backend. The fact setuptools is used in an implementation details and we should try to move away from it, ideally i think the user experience should be the same for building wheels.

The other advantage is that it keeps the source tree clean of any artifact. 

Timing wise, here is quick comparison for this project

```
$ mkvirtualenv -p python3.5 createrepo_c-pip-wheel

$ time pip wheel . -w dist --no-cache-dir
real	0m5.113s
user	0m13.332s
sys	0m1.104s
```

```
$ mkvirtualenv -p python3.5 createrepo_c-setup-bdist-wheel

$ time bash -c "pip install -r requirements-dev.txt && python setup.py bdist_wheel"
real	0m3.947s
user	0m11.996s
sys	0m1.180s
```

Reply:
Fair points.  The only objection I have is that w/ Pip v9 (which is what is currently installed on most distributions, including the newest Fedora and Ubuntu), you would also have to install the ```wheel``` package because that functionality isn't included by default.  It looks like Pip v10+ does include it however.

I would probably err on the side of consistency, so if there's an analogous pip command for building sdists, we can use those.  But otherwise I would probably stick with the "standard" way for now and update it later on.

Reply:
> so if there's an analogous pip command for building sdists

No

> I would probably stick with the "standard" way for now and update it later on.

Agreed.